Procfile

```diff
@@ -3,7 +3,7 @@
 ###############################
 
 # Procfile for development using the new threaded worker (scheduler, twitter stream and delayed job)
-web: bundle exec rails server -b0.0.0.0
+web: bundle exec rails server -p $PORT -b $IP
```

Comment:
Since existing users may not have these variables, we'll need to set defaults here.